specs/phase0/beacon-chain.md

```diff
@@ -773,7 +773,7 @@ def compute_committee(indices: Sequence[ValidatorIndex],
     Return the committee corresponding to ``indices``, ``seed``, ``index``, and committee ``count``.
     """
     start = (len(indices) * index) // count
-    end = (len(indices) * (index + 1)) // count
+    end = (len(indices) * uint64(index + 1)) // count
```

Comment:
I thought this wouldn't be necessary anymore with the changes to `remerkleable`? `uint64 + int -> uint64`?

Reply:
The error message (#1707) was `error: Need type annotation for 'end'`. I think mypy couldn't determine the type here with static checks.

I can change it to `end: uint64 = (len(indices) * (index + 1)) // count` alternatively.

Reply:
Maybe it works when you `uint64(len(indices))` instead? I find it weird to coerce the result, instead of the inputs.